ios/keyman/Keyman/Keyman/ActivityItemProvider.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ActivityItemProvider.swift
+//  Keyman
+//
+//  Created by Gabriel Wong on 2017-09-04.
+//  Copyright © 2017 SIL International. All rights reserved.
+//
+
+import UIKit
+
+let htmlMailFormat =
+  "<html><head><style type=\"text/css\">pre {font-family:\"%@\";font-size:%@;}</style>" +
+  "</head><body><pre>%@</pre>%@</body></html>"
+let mailFooterTextForPad = "<br><br>Sent from&nbsp<a href=\"http://keyman.com/ipad\">Keyman for iPad</a>"
+let mailFooterTextForPhone = "<br><br>Sent from&nbsp<a href=\"http://keyman.com/iphone\">Keyman for iPhone</a>"
+let fbText = "Can't read this? Help at http://keyman.com/fonts"
+
+// Prepares the share text
+class ActivityItemProvider: UIActivityItemProvider {
```

Comment:
OK, so this more directly implements `UIActivityItemProvider` than the old Obj-C implementation, which used a protocol `UIActivityItemSource` to facilitate the implementation.  That explains the differences between the old and new sources pretty well.

https://developer.apple.com/documentation/uikit/uiactivityitemprovider
https://developer.apple.com/documentation/uikit/uiactivityitemsource